test/integration/test_spreadsheet_generator.py

```diff
@@ -12,7 +13,13 @@
 import tempfile
 import yaml
 
+
+# TODO these are integration tests, could either move to a separate directory
```

Comment:
good idea.

Reply:
I removed the comments, I already moved them now to a separate directory to make it easier to run tests locally 😅